flask_appbuilder/baseviews.py

```diff
@@ -1015,6 +1018,23 @@ def _fill_form_exclude_cols(self, exclude_cols, form):
                 field = getattr(form, filter_key)
                 field.data = rel_obj
 
+    def prefill_form(self, form, pk):
+        """
+            Override this, will be called only if the current action is rendering
+            an edit form, and is used to perform additional action to
+            prefill the form.
+
+            This is useful when you have added custom fields that depend on the
+            database contents. Fields that were added by name of a normal column
+            or relationship should work out of the box.
+
+            Example:
```

Comment:
for RST to render the code block to render properly I think you need two colons as in `example::`
https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/baseviews.py#L358

Reply:
gotcha, done.